.github/workflows/core.yml

```diff
@@ -94,97 +97,25 @@ jobs:
           key: ${{ runner.os }}-zeppelin-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-zeppelin-
+      - name: install environment
+        run: ./mvnw install -DskipTests -DskipRat -am -pl .,zeppelin-interpreter,zeppelin-interpreter-shaded,${INTERPRETERS} -Pscala-2.10 -B
```

Comment:
Is there a reason to still support/test Scala 2.10 ?
Most of the projects dropped support even for 2.11

Reply:
You are right, but many interpreter modules depends on scala-2.10, and seems there're no maintainers now. We can raise a discussion in community about dropping these un-maintanced interpreter modules.